src/main.js

```diff
@@ -0,0 +1,16 @@
+import Vue from "vue";
+import App from "./App.vue";
+import router from "./router";
+import store from "./store";
+import axios from "axios";
+import VueAxios from "vue-axios";
+
+Vue.use(VueAxios, axios);
+Vue.config.productionTip = false;
+
+new Vue({
+  router,
+  store,
```

Comment:
Nie używasz store. Jak nie używasz to nie ma sensu trzymać. Zwiększa build i czas budowania niepotrzebnie.